ui/src/dev/server/Dockerfile

```diff
@@ -1,5 +1,29 @@
 FROM node:alpine
 
+#
+# https://github.com/GoogleCloudPlatform/cloud-sdk-docker/blob/master/alpine/Dockerfile
+#
+
+ENV CLOUD_SDK_VERSION 165.0.0
```

Comment:
FYI this is now at 189.0.0. Probably not worth a Jira, but maybe a TODO.

Reply:
Done.